src/features/Landing/index.tsx

```diff
@@ -58,13 +68,27 @@ export const LandingHeader = () => {
         </p>
 
         <span className="hide-print">
-          <UswdsReactLink
-            className="usa-button bg-mint-cool-vivid text-white width-auto"
-            variant="unstyled"
-            to="/signin"
-          >
-            {t('signIn')}
-          </UswdsReactLink>
+          {hasSession ? (
```

Comment:
Similar to the code in the header -- I think a quick comment on the ternary for `hasSession` could be useful!